its/projects/SymbolicLinks/SymbolicLinks/SymbolicLinks.csproj

```diff
@@ -0,0 +1,9 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
While it's usually fine, in this case, I find `its/projects/SymbolicLinks/SymbolicLinks/SymbolicLinks.csproj` over-symbolic-linked in the name.

Can it be `its/projects/SymbolicLinks/Project/Project.csproj` or `its/projects/SymbolicLinks/WithLinks/WithLinks.csproj`, so it is easier to identify what is linking what?

While the project structure itself is fine, reading the Java code is confusing, as it has mix of both.